safe_transaction_service/utils/serializers.py

```diff
@@ -26,3 +28,15 @@ def get_safe_owners(safe_address: ChecksumAddress) -> List[ChecksumAddress]:
         raise ValidationError(
             "Problem connecting to the ethereum node, please try again later"
         )
+
+
+class EpochDateTimeField(DateTimeField):
```

Comment:
Maybe we could move this to `safe-eth-py`

Reply:
I was evaluating the same, and my reasoning is that this is not strictly a case limited to blockchain and for me `safe-eth-py` looks more like a group of tools that helps to interact with Safe and blockchain.

Reply:
Ok, let's reevaluate in the future if we need it in more services